background.js

```diff
@@ -19,23 +19,40 @@ function zeroPad(i) {
 	return i;
 }
 
+function formatDateToTime(date) {
+	return date.toLocaleTimeString()
+}
+
+function withOpenTab(callback) {
+	chrome.tabs.query({
+		active: true,
+		currentWindow: true
+	}, (tabs) => {
+		var tab = tabs[0];
+
+		if (tab) {
+			callback(tab);
+		}
+	});
+}
+
 function updateTrackedEventsForTab(tabId,port) {
 	var sendEvents = [];
-	
+
```

Comment:
probably a formatter issue here, right? worth running through a linter so we're not committing random whitespace